### PR TITLE
Add descriptions to children docs-Studio Pro guide

### PR DESCRIPTION
**Pull request by Yiyun333**
Commit part of the changes for Studio Pro Guide

### CODE REVIEW

File: content/en/docs/refguide/general/third-party-licenses.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: "Third-Party Licenses"
 url: /refguide/third-party-licenses/
 category: "General Info"
 weight: 70
+description: "Describes where to find the available thrid-party licenses in Mendix."
```
Comment (MarkvanMents) [inline content/en/docs/refguide/general/third-party-licenses.md]: typo
```suggestion
description: "Describes where to find the available third-party licenses in Mendix."
```

File: content/en/docs/refguide/modeling/pages/_index.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ title: "Pages"
 url: /refguide/pages/
 category: "App Modeling"
 weight: 30
+description: "Presents an overvies of the features of pages and relevant page resources in Studio Pro."
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/pages/_index.md]: type
```suggestion
description: "Presents an overview of the features of pages and relevant page resources in Studio Pro."
```

File: content/en/docs/refguide/version-control/new-merge-algorithm.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ linktitle: "Merge Algorithm & Conflict Resolution"
 url: /refguide/new-merge-algorithm/
 category: "Version Control"
 weight: 30
+description: "Introduces a new merge algorithm and how to enable it to resolve conflicts."
```
Comment (MarkvanMents) [inline content/en/docs/refguide/version-control/new-merge-algorithm.md]: I think we need a new verb for "how to enable"?
```suggestion
description: "Introduces a new merge algorithm and describes how to enable it to resolve conflicts."
```

File: content/en/docs/refguide/version-control/on-premises-svn.md [hunk 4]
```diff
@@ -3,6 +3,7 @@ title: "Working with SVN On-Premises Version Control Server"
 linktitle: "SVN On-Premises Version Control Server"
 url: /refguide/on-premises-svn/
 weight: 50
+description: "Introduces how to work with SVN on-premises version control server."
```
Comment (MarkvanMents) [inline content/en/docs/refguide/version-control/on-premises-svn.md:6]: Not sure whether this should be "**_an_** SVN on-premises version control server"

But I see that it matches the rest of the document, so let's leave it for now.

(Applies also to the git server - so something to review later).

File: content/en/docs/refguide/runtime/metrics.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 title: "Metrics"
 url: /refguide/metrics/
 category: "Mendix Runtime"
+description: "Describes how to congifure and report metrics in Mendix."
```
Comment (MarkvanMents) [inline content/en/docs/refguide/runtime/metrics.md]: typo
```suggestion
description: "Describes how to configure and report metrics in Mendix."
```

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi @Yiyun333 
Great work. I only have a couple of comments.
Do you have a spell checker in VS Code or whatever you are using to edit? I think that some of the typos would have been picked up by a spell checker. Let me know if you need help in adding a spell checker to your system.
But generally very good and makes the documentation look like it is supposed to and helps the customers find the information they need.
Thanks.

File: content/en/docs/refguide/version-control/on-premises-svn.md (hunk 4 above)
Comment (Yiyun333) [reply content/en/docs/refguide/version-control/on-premises-svn.md:6]: No problem. I will keep it like this for now.

**Comment by Yiyun333** [comment]
> Hi @Yiyun333 Great work. I only have a couple of comments. Do you have a spell checker in VS Code or whatever you are using to edit? I think that some of the typos would have been picked up by a spell checker. Let me know if you need help in adding a spell checker to your system. But generally very good and makes the documentation look like it is supposed to and helps the customers find the information they need. Thanks.

Great tool! Thanks, Mark! I've accepted all the commits you suggested here. After your approval, I will merge this PR :)

**Review by MarkvanMents — APPROVED** [review]
Thanks for picking up my suggestions.
Approving now.
Don't know what you have agreed with Adam about reviewing this PR, but as far as I'm concerned you can merge it.

**Comment by Yiyun333** [comment]
> Thanks for picking up my suggestions. Approving now. Don't know what you have agreed with Adam about reviewing this PR, but as far as I'm concerned you can merge it.

Thanks Mark! As far as I know, Adam is not involved in this PR. So far, it's just you and me.

**Pull request merged**